src/components/Error.jsx

```diff
@@ -0,0 +1,18 @@
+import { Text } from "@mantine/core";
+
+const Error = ({ error }) => {
+  if (!error) {
```

Comment:
If there is no error, I would expect that our error component would not be used, therefore this line of code seems redundant